misc/python/materialize/mzcompose/__init__.py

```diff
@@ -623,15 +624,36 @@ def up(
             self.compose = old_compose
             self._write_compose()
 
-    def down(self, destroy_volumes: bool = True, remove_orphans: bool = True) -> None:
+    def sanity_restart_mz(self) -> None:
+        """Restart Materialized if it exists to find problems with persisted
```

Comment:
~~I don't understand this comment. Did you mean `exits` instead of `exists`? If so, use `terminates` instead?~~
I was first confused by this comment. Change "if it exists" to "if it is running"?

Reply:
Ok